clevis/Sanity/password-generation/test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
I would add here author and license information. Similar to this:
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#
#   test.sh of /CoreOS/clevis/Sanity/password/generation
#   Description: ...
#   Author: Martin...
#
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#
# Copyright (C) 2023  Martin ...
#
# (And the license you consider)
#
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~